tests/tracer/test_ext.py

```diff
@@ -23,5 +26,118 @@ def _updateenv(monkeypatch, env):
 
 @pytest.mark.parametrize("name,environment,tags", _ci_fixtures())
 def test_ci_providers(monkeypatch, name, environment, tags):
+    """Make sure all provided environment variables from each CI provider are tagged correctly."""
     _updateenv(monkeypatch, environment)
-    assert tags == ci.tags(environment), "wrong tags in {0} for {1}".format(name, environment)
+    extracted_tags = ci.tags(environment)
+    for key, value in tags.items():
+        assert extracted_tags[key] == value, "wrong tags in {0} for {1}".format(name, environment)
```

Comment:
Is this change still needed with the new fixture file?

Reply:
Yup, because not all CI provider envs include all relevant git metadata (ex: `git.commit.committer.*`, `git.commit.author.date`).